### PR TITLE
gevent: add recipe

### PR DESCRIPTION
**Pull request by nehaljwani**

### CODE REVIEW

File: recipes/gevent/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - greenlet >=0.4.10
```
Comment (jakirkham) [inline recipes/gevent/meta.yaml]: Should [`cffi`]( https://github.com/gevent/gevent/blob/1.2.2/setup.py#L101 ) be added?

**Comment by jakirkham** [comment]
One question above. Otherwise LGTM.

File: recipes/gevent/meta.yaml (hunk 1 above)
Comment (nehaljwani) [inline recipes/gevent/meta.yaml]: I should be reading `setup.py` too before sending PRs and not rely solely on `conda skeleton` :smile:

**Comment by nehaljwani** [comment]
Skipped cffi on windows since they don't seem to be doing it in their [Appveyor builds ](https://ci.appveyor.com/project/denik/gevent/build/1.0.876/job/r0iqfvg9olfa0urb)either.

**Comment by jakirkham** [comment]
Yeah, seems reasonable to me.

**Comment by jakirkham** [comment]
Thanks @nehaljwani.

**Pull request merged**